libs/cardano-ledger-api/src/Cardano/Ledger/Api/State/Query.hs

```diff
@@ -351,3 +357,21 @@ finishedPulserState ::
   NewEpochState era ->
   (PulsingSnapshot era, RatifyState era)
 finishedPulserState nes = finishDRepPulser (nes ^. newEpochStateGovStateL . drepPulsingStateGovStateL)
+
+-- | Query a stake pool's reward account delegatee which determines the pool's default vote
+-- in absence of an explicit vote. Note that this is different from the delegatee determined
+-- by the credential of the stake pool itself.
+querySPODefaultVote ::
```

Comment:
We call them SPOs in tests and during conversations for the sake of brevity, but  we should not be using anywhere else ledger, since SPO is a person or an organization who **O**perates" a Stake Pool. Everything in ledger in reality is about stake pools, not stake pool operators. So, we should use StakePool in all publicly visible interface, instead of SPO. This point about the name is only about accuracy, but also about consistency.
```suggestion
queryStakePoolDefaultVote ::
```

Reply:
Good point! I'm familiar with the abbreviation but I had the impression that we prefer using SPO (which I never quite liked for the same reasons you point out). It seems like my impression was a false illusion stemming from the fact that I spent a lot of time in tests recently. :D